lib/index.js

```diff
@@ -8,6 +8,14 @@ const miniget = require('miniget');
 const m3u8stream = require('m3u8stream');
 const parseTime = require('m3u8stream/dist/parse-time');
 
+const current = 'v4.0.5';
+const message = '\x1B[31mWARNING:\x1B[0m ytdl-core is out of date! Update it using "npm i ytdl-core".';
```

Comment:
how about yellow to signify more of a warning? red is used more with errors

and i would put `npm install ytdl-core@latest`. `install` to be more specific on what the command does. and `latest` because otherwise npm will install the version that satisfies `package.json`

Reply:
oh, move this to the line below since it's only used there

Reply:
this has been done in the most recent commit